tests/Api/Response/Parser/FiltersParserTest.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+namespace Findologic\Tests\Api\Response\Parser;
+
+use Findologic\Api\Response\Parser\FiltersParser;
+use Findologic\Api\Response\Response;
+use Findologic\Api\Services\Image;
+use Plenty\Modules\Plugin\Libs\Contracts\LibraryCallContract;
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
+
+/**
+ * Class FiltersParserTest
+ * @package Findologic\Tests\Api\Response\Parser
+ */
+class FiltersParserTest extends TestCase
+{
+    /**
+     * @var Image|MockObject
+     */
+    protected $imageService;
+
+    /**
+     * @var LibraryCallContract|MockObject
+     */
+    protected $libraryCallContract;
+
+    public function setUp()
+    {
+        $this->libraryCallContract = $this->getMockBuilder(LibraryCallContract::class)
+            ->disableOriginalConstructor()
+            ->setMethods([])
+            ->getMock();
+        $this->imageService = $this->getMockBuilder(Image::class)
+            ->disableOriginalConstructor()
+            ->setMethods([])
+            ->getMock();
+    }
+
+    public function testParse()
+    {
+        /** @var FiltersParser|MockObject $filtersParserMock */
+        $filtersParserMock = $this->getFiltersParserMock();
+
+        $response = $this->getResponse();
+        $results = $filtersParserMock->parse($response);
+
+        $this->assertNotEmpty($results);
+
+        foreach ($results as $index => $result) {
+            $this->assertArrayHasKey('id', $result);
+            $this->assertArrayHasKey('name', $result);
+            $this->assertArrayHasKey('select', $result);
+            $this->assertArrayHasKey('type', $result);
+            $this->assertArrayHasKey('values', $result);
+
+            $this->assertEquals($result['id'], $response->filters->filter[$index]->name);
+            $this->assertEquals($result['name'], $response->filters->filter[$index]->display);
+            $this->assertEquals($result['select'], $response->filters->filter[$index]->select);
+            $this->assertEquals($result['type'], $response->filters->filter[$index]->type);
+            $this->assertEquals(count($result['values']), count($response->filters->filter[$index]->items->item));
```

Comment:
There should be an assertion method available explicitly for this case. I think it is called something like `assertCount`. Try to use that instead of `assertEquals`.